huaweicloud/services/rds/resource_huaweicloud_rds_mysql_account.go

```diff
@@ -91,85 +114,144 @@ func resourceRdsAccountCreate(ctx context.Context, d *schema.ResourceData, meta
 		return nil
 	})
 	if err != nil {
-		return diag.Errorf("error creating RDS database user: %s", err)
+		return diag.Errorf("error creating RDS Mysql account: %s", err)
 	}
 
-	id := instanceId + "/" + dbUser
-	d.SetId(id)
-	return resourceRdsAccountRead(ctx, d, meta)
+	instanceId := d.Get("instance_id").(string)
+	accountName := d.Get("name").(string)
+	d.SetId(instanceId + "/" + accountName)
+
+	return resourceMysqlAccountRead(ctx, d, meta)
+}
+
+func buildCreateMysqlAccountBodyParams(d *schema.ResourceData) map[string]interface{} {
+	bodyParams := map[string]interface{}{
+		"name":     utils.ValueIngoreEmpty(d.Get("name")),
+		"password": utils.ValueIngoreEmpty(d.Get("password")),
+		"comment":  utils.ValueIngoreEmpty(d.Get("description")),
+		"hosts":    utils.ValueIngoreEmpty(d.Get("hosts")),
+	}
+	return bodyParams
 }
 
-func resourceRdsAccountRead(ctx context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
-	c := meta.(*config.Config)
-	client, err := c.HcRdsV3Client(c.GetRegion(d))
+func resourceMysqlAccountRead(_ context.Context, d *schema.ResourceData, meta interface{}) diag.Diagnostics {
+	cfg := meta.(*config.Config)
+	region := cfg.GetRegion(d)
+
+	var mErr *multierror.Error
+
+	// getMysqlAccount: query RDS Mysql account
+	var (
+		getMysqlAccountHttpUrl = "v3/{project_id}/instances/{instance_id}/db_user/detail"
+		getMysqlAccountProduct = "rds"
+	)
+	getMysqlAccountClient, err := cfg.NewServiceClient(getMysqlAccountProduct, region)
 	if err != nil {
-		return fmtp.DiagErrorf("error creating RDS client: %s", err)
+		return diag.Errorf("error creating RDS client: %s", err)
 	}
 
 	// Split instance_id and user from resource id
-	parts := strings.SplitN(d.Id(), "/", 2)
+	parts := strings.Split(d.Id(), "/")
 	if len(parts) != 2 {
-		return fmtp.DiagErrorf("invalid id format, must be <instance_id>/<user>")
+		return diag.Errorf("invalid id format, must be <instance_id>/<name>")
 	}
 	instanceId := parts[0]
-	dbUser := parts[1]
+	accountName := parts[1]
+
+	getMysqlAccountBasePath := getMysqlAccountClient.Endpoint + getMysqlAccountHttpUrl
+	getMysqlAccountBasePath = strings.ReplaceAll(getMysqlAccountBasePath, "{project_id}",
+		getMysqlAccountClient.ProjectID)
+	getMysqlAccountBasePath = strings.ReplaceAll(getMysqlAccountBasePath, "{instance_id}", instanceId)
 
-	// items on every page, [1, 100]
-	limit := int32(100)
-	// List all db users
-	request := &model.ListDbUsersRequest{
-		InstanceId: instanceId,
-		Limit:      limit,
-		Page:       int32(1),
+	page := 1
```

Comment:
maybe you can declare a query parameter: `page`, then will auto-generate those code

Reply:
done